tests/saitests/sai_qos_tests.py

```diff
@@ -2119,48 +2119,64 @@ def runTest(self):
                 if actual_pkts_num_leak_out > pkts_num_leak_out:
                     send_packet(self, src_port_id, pkt, actual_pkts_num_leak_out - pkts_num_leak_out) 
 
-            q_wm_res, pg_shared_wm_res, pg_headroom_wm_res = sai_thrift_read_port_watermarks(self.client, port_list[dst_port_id])
-            print >> sys.stderr, "Init pkts num sent: %d, min: %d, actual watermark value to start: %d" % ((pkts_num_leak_out + pkts_num_fill_min), pkts_num_fill_min, q_wm_res[queue])
-            if pkts_num_fill_min:
-                assert(q_wm_res[queue] == 0)
+            if asic_type == 'cisco-8000':
+                pkt_count_list = [50,1000,4000,7000,12500]
```

Comment:
Not fully understand what those pkt count mean, could you give some more comments?